Update terraform-vm-msi.md

VaijanathB opened the pull request:
Based on the comments, users seem to misinterpret this line. Editing this to make it more clearer.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @msfussell, has been notified to review your proposed change.

msfussell commented:
@alexandair. Thank you for this update.
@tysonn Looks good please merge this change,

VaijanathB committed:
Update terraform-vm-msi.md

Based on the comments, users seem to misinterpret this line. Editing this to make it more clearer.
